### ISSUE TITLE
Specify number of scales sold at trading post

### ISSUE THREAD

**Issue by CrazyPython** [doublespeakgames/adarkroom#448]
Probably an easy patch; if I have time I might provide one (probably won't).

**Comment by grotewold** [comment]
Not exactly sure what you mean by this. You can buy as many scales as you want. or are you talking about storing how many you have bought over the course of your game?

**Comment by CrazyPython** [comment]
At the trading post, it just says "scales" for the button to buy scales. I tried it once, and I only got one scale. Why not change the text to say "one scale" instead?

**Comment by grotewold** [comment]
Yeah, its a little confusing, but it is a handful of scales, and when you buy it, you have 1 handful of scales. When you have 50 scales it's actually 50 bunches of scales. so every time you buy scales, you'll get 1 scales. Adding the quantity is confusing to say, "1 scales" and lengthening it to "handful of scales" is too long. Does that make sense? I'm not sure what the best answer is.

**Comment by vankasteelj** [comment]
imagine you want to buy a bayonet because you died like a noob (true story), but the amount of scales required is really high, you don't have enough. But you do have fur. 

For the sake of my point, i'll call 'scales' ITEM1 and 'fur' ITEM2

You'll need 500 (I think, it doesn't really matter) ITEM1 to buy a bayonet, but you only have 400. You'll trade 150 ITEM2 to get 1 ITEM1. and once you have enough ITEM1, you can buy your bayonet.

I don't see anything misleading here, because the fact that ITEM1 is 'a scale' or 'a handful of scales' doesn't matter, as you need a fixed number of it anyway.

Apart from the clearly terrible deal you're making (150 furs for it), it seems pretty clear to me. 

Maybe what CrazyPython meant is that you can't know how many ITEM1 you'll get for 150 ITEM2. That's what I felt when I was playing, I was thinking i'd get more than only one ITEM1 for 150 ITEM2. 

As the game is heavily resting on tooltips, it might be worth considering changing the trade tooltip, maybe like Heroes of Might and Magic does it, by printing an 'exchange ratio', like `fur    150:1` instead of the current `fur    150`

**Comment by AndySky21** [comment]
I have to answer this.
Some weeks ago (several) I thought about a way to buy more than one item
through a slider. This would also show the user how many units they are
buying.
Unfortunately, my version of ADR is heavily modified on code side, so
before I post that I have to check if my changes work perfectly. I'm going
to post the code update before April ends and the slider soon after.

**Comment by gitupofftheflooranddosomework** [comment]
please close, this appears to be a text issue that was explained to not be an issue
